### PR TITLE
Serbian Cyrilic and Hungarian language localization

### PR DESCRIPTION
**Pull request by ptodic**

### CODE REVIEW

File: hu.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+"STATION_DIRECTORY_STATIONS_SEARCH_PLACEHOLDER" = "Állomások keresése";
+
+"ADD_STATION_CONTEXT_ADD_STATION" = "Kézi hozzáadás";
+"ADD_STATION_CONTEXT_SHOW_DIRECTORY" = "Listából hozzáad";
+
+"LIBRARY_STATE_UPLOADING" = "Feltöltés…";
+"LIBRARY_STATE_DOWNLOADING" = "Letöltés…";
+"LIBRARY_STATE_PREPARING" = "Előkészítés…";
+"LIBRARY_STATE_READY" = "Naprakész";
+"LIBRARY_STATE_MIGRATING" = "Migrálás…;
```
Comment (steventroughtonsmith) [inline hu.lproj/Localizable.strings:140]: Missing a closing " here

**Comment by steventroughtonsmith** [comment]
Hey @ptodic; now that I'm preparing the next update to Broadcasts I'd love to include these. Let's verify one at a time — how does the Hungarian look to you here?

Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-03 at 22 50 07
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-03 at 22 50 11
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-03 at 22 50 19
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-03 at 22 50 25
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-03 at 22 50 28
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-03 at 22 50 32
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-03 at 22 50 35
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-03 at 22 50 41

**Comment by ptodic** [comment]
Looks like it is good to go :) 2 small issues fixed.

**Comment by steventroughtonsmith** [comment]
OK, great. And now how does the Serbian look?

Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-04 at 15 20 13
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-04 at 15 20 18
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-04 at 15 20 25
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-04 at 15 20 36
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-04 at 15 20 39
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-04 at 15 20 45
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-04 at 15 20 46
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-06-04 at 15 20 52

**Comment by ptodic** [comment]
Serbian looks good!

**Comment by steventroughtonsmith** [comment]
Awesome, thanks! I will roll these both into the next update to Broadcasts (1.3.6). 😄

**Pull request merged**